### ISSUE TITLE
R: capabilities("tcltk") on Wynton's Rocky 8 nodes

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/CBI-software#110]
# Issue

```sh
$ Rscript -e 'capabilities("tcltk")'
tcltk 
FALSE
```

```diff
$ diff -u -w /wynton/home/cbi/shared/software/CBI/_centos7/R-4.3.1-gcc10/config.log.summary /wynton/home/cbi/shared/software/CBI/_rocky8/R-4.3.1-gcc10/config.log.summary 
--- /wynton/home/cbi/shared/software/CBI/_centos7/R-4.3.1-gcc10/config.log.summary	2023-09-07 17:12:54.000000000 -0700
+++ /wynton/home/cbi/shared/software/CBI/_rocky8/R-4.3.1-gcc10/config.log.summary	2023-09-15 17:07:27.000000000 -0700
@@ -1,7 +1,7 @@
 R is now configured for x86_64-pc-linux-gnu
 
   Source directory:            .
-  Installation directory:      /wynton/home/cbi/shared/software/CBI/_centos7/R-4.3.1-gcc10
+  Installation directory:      /wynton/home/cbi/shared/software/CBI/_rocky8/R-4.3.1-gcc10
 
   C compiler:                  gcc  -g -O2
   Fortran fixed-form compiler: gfortran  -g -O2
@@ -15,7 +15,7 @@
   Fortran free-form compiler:  gfortran  -g -O2
   Obj-C compiler:	        
 
-  Interfaces supported:        X11, tcltk
+  Interfaces supported:        X11
   External libraries:          pcre2, readline, curl
   Additional capabilities:     PNG, JPEG, TIFF, NLS, cairo, ICU
   Options enabled:             shared R library, shared BLAS, R profiling, memory profiling
```

# Troubleshooting

Rocky 8:

```sh
[dev3]$ grep -A 1 -E "checking for (tcl|tk)" /wynton/home/cbi/shared/software/CBI/_rocky8/R-4.3.1-gcc10/config.log
configure:50048: checking for tclConfig.sh
configure:50086: result: no
configure:50095: checking for tclConfig.sh in library (sub)directories
configure:50117: result: no
configure:50129: checking for tkConfig.sh
configure:50167: result: no
configure:50176: checking for tkConfig.sh in library (sub)directories
configure:50198: result: no
configure:50295: checking for tcl.h
conftest.c:271:10: fatal error: tcl.h: No such file or directory
```

CentOS 7:

```sh
[dev1]$ grep -A 1 -E "checking for (tcl|tk)" /wynton/home/cbi/shared/software/CBI/_centos7/R-4.3.1-gcc10/config.log
configure:50048: checking for tclConfig.sh
configure:50086: result: no
configure:50095: checking for tclConfig.sh in library (sub)directories
configure:50114: result: /usr/lib64/tclConfig.sh
configure:50129: checking for tkConfig.sh
configure:50167: result: no
configure:50176: checking for tkConfig.sh in library (sub)directories
configure:50195: result: /usr/lib64/tkConfig.sh
configure:50259: checking for tcl.h
configure:50259: gcc -c  -g -O2 -fpic -I/usr/local/include  -I/usr/include conftest.c >&5
```

# Conclusion

* [x] `tcl-devel` is missing on Rocky 8.
* [x] `tk-devel` is missing on Rocky 8.

**Comment by HenrikBengtsson** [comment]
Fixed R 4.3.1 on Rocky 8.

Remaining task is to reinstall R for the older version on Rocky 8.

**Comment by HenrikBengtsson** [comment]
https://cran.r-project.org/src/base/R-4/:

* [ ] R 4.0.0
* [ ] R 4.0.5
* [ ] R 4.1.0
* [ ] R 4.1.3
* [ ] R 4.2.0
* [x] R 4.2.3
* [ ] R 4.3.0
* [x] R 4.3.1
* [x] R 4.3.2

**Comment by HenrikBengtsson** [comment]
Solved for R (>= 3.3.3).   Older versions of R has other installation issues on Rocky 8, which are now tracked in Issue #116.
